package.json

```diff
@@ -43,6 +43,7 @@
     "@testing-library/react": "11.2.2",
     "@testing-library/user-event": "12.2.2",
     "@types/auth0-js": "9.14.2",
+    "@types/classnames": "^2.2.11",
```

Comment:
nit: pin

Reply:
Ugh, I thought I added configuration to stop this.